### ISSUE TITLE
PHPDoc wrong return type for Friendable::findFriendships

### ISSUE THREAD

**Issue by FlYos** [multicaret/laravel-acquaintances#96]
Hi,

If I am not mistaken, I believe the PHPDoc is wrong for the return type. 
It should be `Illuminate\Database\Eloquent\Builder` instead of `Illuminate\Database\Eloquent\Collection`, It's right ?

https://github.com/multicaret/laravel-acquaintances/blob/74038bc4b9cca6d4d6e6176cecff6e16fc4a5faf/src/Traits/Friendable.php#L428-L467

**Comment by mkwsra** [comment]
Yes indeed, please give it a run on your locale machine after you change the libraries file from within `/vendor` and if everything is okay, can you please create a PR? I would be glad to accept it.

Thanks for noticing this
